scripts/release.py

```diff
@@ -96,7 +96,7 @@ def create_release_draft():
                 continue
             if other_patch_num > latest_patch_version:
                 latest_patch_version = other_patch_num
-        new_patch_version = latest_patch_version + 1
+        new_patch_version = latest_patch_version if latest_patch_version == 1 else latest_patch_version + 1
```

Comment:
@ZStriker19 , what was this PR fixing?

If we're currently at patch `1.18.1`, then `new_patch_version` should become `2`.

With this change, if `latest_patch_version` starts at `1`, and `other_patch_num` will also be `1`, so `latest_patch_version` will remain `1` because `latest_patch_version == 1`.